lib/http/request.js

```diff
@@ -44,9 +50,8 @@ Request.prototype.toString = function() {
 
   var params = '';
   if (this.params && this.params !== this.ANY) {
-    params = '?' + _.join(_.chain(_.keys(this.params))
-      .map(function(key) { return key + '=' + this.params[key]; }.bind(this))
-      .value(), '&');
+    params = '?' + Object.keys(this.params)
+      .map(key => function() { return key + '=' + this.params[key]; }.bind(this)).join('&');
```

Comment:
The function defined in .map is not executed, so each map function returns a function. This will return ?<function>&<function>.

fix: add () after .bind(this)

Object.keys(this.params).map(key => function() { return key + '=' + this.params[key]; }.bind(this)()).join('&');

Reply:
`'?' + Object.keys(v).map(key => key + '=' + v[key]).join('&');` this seems like it would work too but I'm unsure if there are corner cases